src/pages/UserAccountPage.js

```diff
@@ -41,6 +41,8 @@ import withModal from 'components/WithModal';
 import CubePreview from 'components/CubePreview';
 import CubePropType from 'proptypes/CubePropType';
 
+const LEVELS = ['Patron', 'Cobra Hatchling', 'Coiling Oracle', 'Lotus Cobra'];
```

Comment:
Just as an ex-post minor note, I would have liked some comment here saying this mirrors the definition from the model (and vice versa), but it's not really a big deal